docs/SplunkOperatorUpgrade.md

```diff
@@ -0,0 +1,77 @@
+# How to upgrade Splunk Operator and Splunk Enterprise Deployments
+​
+To upgrade the Splunk Operator for Kubernetes, you will overwrite the prior operator release with the latest version. Once the latest version of the Splunk Operator is running, the CRD's are updated and any running pods in the operator managed namespace are restarted. Any new spec defined by the operator will be applied to the restarted pods.
+​
+A Splunk Operator for Kubernetes upgrade might include support for a later version of the Splunk Enterprise Docker image. After the Splunk Operator completes its upgrade, the pods will be restarted using the latest Splunk Enterprise Docker image.
+​
+* Note: The Splunk Operator does not provide a way to downgrade to a previous release.
+​
+## Prequisites
+​
+* Before you upgrade, review the Splunk Operator [change log](https://github.com/splunk/splunk-operator/releases) page for information on changes made in the latest release. The Splunk Enterprise Docker image compatibility is noted in each release version.
+​
+* If the Splunk Enterprise Docker image changes, review the Splunk Enterprise [Upgrade Readme](https://docs.splunk.com/Documentation/Splunk/latest/Installation/AboutupgradingREADTHISFIRST) page before upgrading. 
+​
+* For general information about Splunk Enterprise compatibility and the upgrade process, see [How to upgrade Splunk Enterprise](https://docs.splunk.com/Documentation/Splunk/latest/Installation/HowtoupgradeSplunk).
+​
+* If you use forwarders, verify the Splunk Enterprise version compatibility with the forwarders in the [Compatibility between forwarders and Splunk Enterprise indexers](https://docs.splunk.com/Documentation/Forwarder/latest/Forwarder/Compatibilitybetweenforwardersandindexers) documentation.
+​
+## Upgrading Splunk Operator and Splunk Operator Deployment
+​
+1. Download the latest Splunk Operator installation yaml file.
+​
+```
+wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/0.2.2/splunk-operator-install.yaml 
+```
+​
+2. (Optional) Review the file and update it with your specific customizations used during your install. 
+​
+3. Upgrade the Splunk Operator.
+​
+```
+kubectl apply -f splunk-operator.yaml
+```
+​
+After applying the yaml, a new operator pod will be created and the existing operator pod will drop. Example:
```

Comment:
Nit: **`will be terminated`** vs **`will drop`**?

Reply:
That wording was suggested by @eddikay. The consensus is to use terminated @eddikay  can you please confirm its acceptable?

Reply:
Will be changing to **terminated** as that is the official term by kubernetes.